presto-spi/src/main/java/com/facebook/presto/spi/RecordCursor.java

```diff
@@ -39,6 +39,8 @@
 
     Slice getSlice(int field);
 
+    Object getObject(int field);
```

Comment:
Ask @dain if he'd like to return T, like `Block.getObject()` does. It seems like it would make sense to me